extension/src/popup/components/manageAssets/SelectAssetRows/index.tsx

```diff
@@ -54,62 +53,61 @@ export const SelectAssetRows = ({
     assetSelect.type === AssetSelectType.PATH_PAY &&
     assetSelect.isSource === false;
 
-    return (
-      <SimpleBar
-        className="SelectAssetRows__scrollbar"
-        style={{
-          maxHeight: `${maxHeight}px`,
-        }}
-      >
-        <div className="SelectAssetRows__content">
-          {assetRows.map(({ code, domain, image, issuer }) => {
-            const assetDomain = assetDomains[getCanonicalFromAsset(code, issuer)];
-            const isScamAsset = !!blockedDomains.domains[assetDomain];
-            
-            return (
-              <div
-                className="SelectAssetRows__row selectable"
-                key={getCanonicalFromAsset(code, issuer)}
-                onClick={() => {
-                  if (assetSelect.isSource) {
-                    dispatch(saveAsset(getCanonicalFromAsset(code, issuer)));
-                    history.goBack();
-                  } else {
-                    dispatch(
-                      saveDestinationAsset(getCanonicalFromAsset(code, issuer)),
-                    );
-                    history.goBack();
-                  }
-                }}
-              >
-                <AssetIcon
-                  assetIcons={
-                    code !== "XLM"
-                      ? { [getCanonicalFromAsset(code, issuer)]: image }
-                      : {}
-                  }
-                  code={code}
-                  issuerKey={issuer}
-                />
-                <div className="SelectAssetRows__row__info">
-                  <div className="SelectAssetRows__row__info__header">
-                    {code}
-                    <ScamAssetIcon isScamAsset={isScamAsset} />
-                  </div>
-                  <div className="SelectAssetRows__domain">
-                    {formatDomain(domain)}
-                  </div>
+  return (
+    <SimpleBar
+      className="SelectAssetRows__scrollbar"
+      style={{
+        maxHeight: `${maxHeight}px`,
+      }}
+    >
+      <div className="SelectAssetRows__content">
+        {assetRows.map(({ code, domain, image, issuer }) => {
+          const isScamAsset = !!blockedDomains.domains[domain];
```

Comment:
we could also make that simplification here (though really doesn't matter since the domains here are for assets we own)